### PR TITLE
Support groups/labels in counters

### PR DESCRIPTION
**Pull request by slvwolf**

### CODE REVIEW

File: src/main/java/com/ccentral4j/CCEtcdClient.java [hunk 1]
```diff
@@ -262,9 +262,17 @@ public void refresh() {
...
   }
 
-  private Counter getCounter(String key) {
-    refresh();
-    key = filterKey(key);
+  private Counter getCounter(String key, String ...groups) {
+    if (groups.length > 0) {
+      StringBuilder b = new StringBuilder(filterKey(key));
+      for (String group : groups) {
+        b.append(".");
```
Comment (shusso) [inline src/main/java/com/ccentral4j/CCEtcdClient.java:269]: would calling `getCounters` with e.g. `getCounters(key, 'my.group.1', 'my.group.2')` be supported or should we strip away possible `.` from the group-name?

File: src/test/java/com/ccentral4j/CCentralTest.java [hunk 2]
```diff
@@ -98,6 +103,16 @@ public void pullConfigLate() throws EtcdAuthenticationException, TimeoutExceptio
     verify(client).fetchConfig();
   }
 
+  /** Increment with groups */
+  @Test
+  public void incGroups() throws Exception {
+    cCentral.incrementInstanceCounter("key", "group1", "group2");
```
Comment (shusso) [inline src/test/java/com/ccentral4j/CCentralTest.java:109]: should we also add a test to make sure the old-behaviour would still work as expected e.g. calling with
```
cCentral.incrementInstanceCounter("key", "value");
```

Comment (slvwolf) [reply src/test/java/com/ccentral4j/CCentralTest.java:109]: Additional test added

File: src/main/java/com/ccentral4j/CCEtcdClient.java (hunk 1 above)
Comment (slvwolf) [reply src/main/java/com/ccentral4j/CCEtcdClient.java:269]: Group values are stripped from all invalid characters and whitespaces are changed into underscores. Added additional test for this as well.

**Review by shusso — APPROVED** [review]
👍

**Pull request merged**